packages/odyssey-react/src/components/Banner/Banner.module.scss

```diff
@@ -0,0 +1,114 @@
+/*!
+ * Copyright (c) 2018-present, Okta, Inc. and/or its affiliates. All rights reserved.
+ * The Okta software accompanied by this notice is provided pursuant to the Apache License, Version 2.0 (the "License.")
+ *
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0.
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *
+ * See the License for the specific language governing permissions and limitations under the License.
+ */
+
+.banner {
+  display: grid;
+  position: relative;
+  grid-template-areas: 'icon title content actions';
```

Comment:
Nice QoL benefit of module naming, our grid areas can match our part class names. 👏🏼

Reply:
See my comment over here in my spike on removing Heading global styles. Would be curious to get your thoughts @edburyenegren-okta https://github.com/okta/odyssey/pull/1032/files#r687225098